salsa_webqa/salsa_runner.py

```diff
@@ -156,14 +173,26 @@ def run_on_browserstack(self):
 
     def run_locally(self):
         """ Runs tests on local browser """
-        self.cleanup_results()
         print('Running for browser: ' + self.driver_name)
         return self.trigger_pytest(self.driver_name)
 
+    def run_non_selenium(self):
+        """ Runs non_selenium_tests"""
+        print('Running non selenium tests')
+        return self.trigger_pytest(None)
+
     def trigger_pytest(self, config_section):
```

Comment:
how about making `config_section` parameter optional: `def trigger_pytest(self, config_section=None):` ?
Then you could trigger non_selenium tests by running just `self.trigger_pytest()` and perhaps even get rid of `run_non_selenium` method?